src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+
+    protected String to;
+    protected String from;
+
+    public Event(String description, String to, String from) {
+        super(description);
+        this.to = to;
+        this.from = from;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " project meeting (from: " + from + " to: " + to + ")";
+    }
+}
```

Comment:
Could consider inheriting from Deadline as the attribute "from" is already there. Inheriting from Task would mean you have to generate another from attribute

Reply:
I have read all your comments. They seems all reasonable. Thanks for your advice and I would improve my code as you said.